custom_conf.py

```diff
@@ -212,6 +213,32 @@
 # that you want to link to:
 # manpages_url = "https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html"
 
+############################################################
+### -- Options for PDF output ------------------------------
+############################################################
+
+latex_additional_files = [
+    "./.sphinx/fonts/Ubuntu-B.ttf",
+    "./.sphinx/fonts/Ubuntu-R.ttf",
+    "./.sphinx/fonts/Ubuntu-RI.ttf",
+    "./.sphinx/fonts/UbuntuMono-R.ttf",
+    "./.sphinx/fonts/UbuntuMono-RI.ttf",
+    "./.sphinx/fonts/UbuntuMono-B.ttf",
+    "./.sphinx/images/Canonical-logo-4x.png",
+    "./.sphinx/images/front-page-light.pdf",
+    "./.sphinx/images/normal-page-footer.pdf",
+]
+
+latex_engine = 'xelatex'
+latex_show_pagerefs = True
+latex_show_urls = 'footnote'
+
+with open(".sphinx/latex_elements_template.txt", "rt") as file:
+    latex_config = file.read()
+
+latex_elements = ast.literal_eval(latex_config.replace("$PROJECT", project))
```

Comment:
I still think most (if not all) of this should go into `conf.py`.
We don't want teams to change the fonts or images, or use a different engine.

I won't block the PR because of it though, since the same discussion will come up again for the "new" starter pack and things can be moved out later as well.

Reply:
I'm happy to move this to the `conf.py` file.

Reply:
Done with [9e0a32e](https://github.com/canonical/sphinx-docs-starter-pack/pull/264/commits/9e0a32ecedc9022d4174d8d57a3308820b5b8f2e)